src/modules/addReviewConfidence.js

```diff
@@ -1,28 +1,72 @@
-//not useful?
-function addReviewConfidence(){
-	generalAPIcall("query{Page(page:1,perPage:30){reviews(sort:ID_DESC){id rating ratingAmount}}}",{},function(data){
-		let adder = function(){
-			if(location.pathname !== "/reviews"){
-				return
-			}
-			let locationForIt = document.querySelector(".recent-reviews .review-wrap");
-			if(!locationForIt){
-				setTimeout(adder,200);
+exportModule({
+	id: "reviewConfidence",
+	description: "Add confidence scores to reviews",
+	isDefault: true,
+	categories: ["Browse"],
+	visible: true,
+	urlMatch: function(url){
+		return url === "https://anilist.co/reviews"
```

Comment:
Fails for ``https://anilist.co/reviews/``. Bug also present in original code.

Reply:
fixed